.github/gitlint/contrib_description_conventional_commits.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from gitlint.rules import CommitRule, RuleViolation
+
+
+class ConventionalCommitsDescription(CommitRule):
+    """ This rule will enforce that each commit description starts with
+    a lowercase.
+    """
+
+    id = "UC1"
+    name = "contrib-description-conventional-commits"
+
+    def validate(self, commit):
+        if ':' not in commit.message.title:
+            self.log.debug("Title does not follow conventional commit specification")
+            return
```

Comment:
I suspect this rule wants to impose the idea of overloading the git commit subject line with metadata?

Can you briefly explain what the subject line would ideally look like using this rule? Maybe paste a few examples?

The reason I ask, is because I want to ensure the subject-lines stay high quality:
- have enough room to use the imperative mood inside 80'ish characters, and 
- not get over-loaded with metadata that could potentially go elsewhere in the git commit message body.

https://en.wikipedia.org/wiki/Imperative_mood

If the idea is to have something like:
- `Bug fix: stop doing that thing that broke something`
- `Enhancement: implement cool new feature`

Then I'm cool..

But, if the idea is to inject Jira issues into the subject-line, then not cool...

Instead, I would suggest adding a rule to parse for jira `project-id` string in the commit message, something like:
```
New feature: imperative mood subject-line

Paragraphs of message text
. . .

jira: hacbs-99999

By-Lines: Captain Crepe <cpt.crepe@example.com>
...
```

https://wiki.openstack.org/wiki/GitCommitMessages#Information_in_commit_messages

Reply:
Specification: https://www.conventionalcommits.org/en/v1.0.0/